### PR TITLE
Query filter (RN 36822+36832)

### PR DESCRIPTION
**Pull request by EdithLansens**

### CODE REVIEW

File: user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Feeds/DashboardQueryFilterFeed.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
+---
+uid: DashboardQueryFilterFeed
+---
+
+# Query filter feed
+
+> [!WARNING]
+> This feature is in preview until DataMiner 10.3.9/10.4.0. If you use the preview version of the feature, its functionality may be different from what is described below. For more information, see [Soft-launch options](xref:SoftLaunchOptions).
+
+Available from DataMiner 10.3.9/10.4.0 onwards. Prior to this, the component is available in soft launch from DataMiner 10.0.4 onwards, if the soft-launch option ReportsAndDashboardsPTP is enabled.
```
[the remark this answers is not in the document]
Comment (EdithLansens) [reply user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Feeds/DashboardQueryFilterFeed.md:10]: "Soft-launch" only requires a hyphen if it is used as an adjective, preceding a noun, e.g. a soft-launch option. The spelling of soft launch without a hyphen is correct here.

File: user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Feeds/DashboardQueryFilterFeed.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+
+  - Select the *Toggle free form* button to display a text box in which you can type a value.
+
+  - Select the *Toggle checklist* button to list all possible values as a checklist.
+
+- When exclusively filtering a node edge graph by node, edges will only be highlighted when both the source and destination are highlighted. Alternatively, when exclusively filtering a node edge graph by edge, the source and/or destination attached to the highlighted edge segments will be highlighted.
+
+## Filtering a table or node edge graph component
+
+When linked to a component using a feed, the query filter component allows real-time filtering of that associated component.
```
Comment (EdithLansens) [inline user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Feeds/DashboardQueryFilterFeed.md:46]: Is this correct, @TVincke98?

File: user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Feeds/DashboardQueryFilterFeed.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+- [Feeding query columns as filter](#feeding-query-columns-as-filter)
+
+### Feeding queries as data
+
+Following setup and configuration of the query filter component:
+
+1. Add a table component to your dashboard.
+
+   > [!NOTE]
+   > In this example, we are using a table component. However, any component capable of receiving a query can be linked to the query filter.
```
Comment (EdithLansens) [inline user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Feeds/DashboardQueryFilterFeed.md:61]: I have added this disclaimer so that it is clear that the table component is only one example of a component that can be linked to a query filter, @TVincke98.

File: user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Feeds/DashboardQueryFilterFeed.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+   > [!NOTE]
+   > The assigned feed name will be visible in the bottom-right corner of the query filter component (e.g. Query filter 1).
+
+1. Optionally, fine-tune the component layout. See [Customizing the component layout](xref:Configuring_dashboard_components#customizing-the-component-layout).
+
+1. Optionally, customize the following component options in the *Component* > *Settings* tab:
+
+   - *WebSocket settings*: Allows you to customize the polling interval for this component. To do so, clear the checkbox in this section and specify the custom polling interval.
+
+   - *Filter assistance*: Determines whether the column filters will assist you by providing suggestions and other information such as units, step size, etc. By default disabled.
```
Comment (EdithLansens) [inline user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Feeds/DashboardQueryFilterFeed.md:27]: This description now directly aligns with the description provided in the Dashboards app.

**Review by EdithLansens — COMMENTED** [review]
Thank you for your feedback, @TVincke98. I have made some final changes based on your comments. Could you take a look?

File: user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Feeds/DashboardQueryFilterFeed.md [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+
+     > [!NOTE]
+     > Depending on the size of your data, enabling this setting may impact performance.
+
+   - *Allow color mode*: Determines whether [conditional coloring](#conditional-coloring) can be applied within the query filter. By default enabled.
+
+- In the top-right corner of the query filter component, you have an *Active (x)* toggle button. If you enable this button, the component displays only the active filter options, and the button itself indicates the number of active options.
```
Comment (MariekeGO) [inline user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Feeds/DashboardQueryFilterFeed.md:34]:
```suggestion
- In the top-right corner of the query filter component, there is an *Active (x)* toggle button. If you enable this button, the component displays only the active filter options, and the button itself indicates the number of active options.
```

File: user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Feeds/DashboardQueryFilterFeed.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+- Next to columns containing discrete values of type string or number, you can find a button that allows you to change how the possible values are displayed when you hover your mouse over it:
+
+  - Select the *Toggle free form* button to display a text box in which you can type a value.
+
+  - Select the *Toggle checklist* button to list all possible values as a checklist.
+
+- When exclusively filtering a node edge graph by node, edges will only be highlighted when both the source and destination are highlighted. Alternatively, when exclusively filtering a node edge graph by edge, the source and/or destination attached to the highlighted edge segments will be highlighted.
```
Comment (MariekeGO) [inline user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Feeds/DashboardQueryFilterFeed.md:42]:
```suggestion
- When a node edge graph is exclusively filtered by node, edges will only be highlighted when both the source and destination are highlighted. Alternatively, when a node edge graph is exclusively filtered by edge, the source and/or destination attached to the highlighted edge segments will be highlighted.
```

File: user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Feeds/DashboardQueryFilterFeed.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+   > [!NOTE]
+   > In this example, we are using a table component. However, any component capable of receiving a query can be linked to the query filter.
+
+1. In the *Data* tab, go to *All available data > Feeds*. Expand the feed linked to the query filter and drag *Queries* onto the table component.
+
+   Each time you change the query filter, a new query is fed to the table, displaying only rows that match the filter settings.
+
+### Feeding query columns as filter
+
+Following setup and configuration of the query filter component:
```
Comment (MariekeGO) [inline user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Feeds/DashboardQueryFilterFeed.md:69]: Could you rephrase this?

File: user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Feeds/DashboardQueryFilterFeed.md [hunk 8]
```diff
@@ -0,0 +1,92 @@
...
+
+Two methods exist for linking a query filter:
+
+- [Feeding queries as data](#feeding-queries-as-data)
+
+- [Feeding query columns as filter](#feeding-query-columns-as-filter)
+
+### Feeding queries as data
+
+Following setup and configuration of the query filter component:
```
Comment (MariekeGO) [inline user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Feeds/DashboardQueryFilterFeed.md:56]: Could you rephrase this?

File: user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Feeds/DashboardQueryFilterFeed.md [hunk 9]
```diff
@@ -0,0 +1,92 @@
...
+
+1. In the *Data* tab, go to *All available data > Feeds*. Expand the feed linked to the query filter and drag *Query columns* onto the yellow filter drop area of the table component.
+
+   Each time you change the query filter, the data inside the table is filtered according to the filter settings in the query filter. No new query is fed to the table. The latter keeps displaying all rows, but those that do not match the filter will turn gray.
+
+## Conditional coloring
+
+If the *Allow color mode* setting is enabled, a color marker icon will be available in the top-right corner of the filter query component.
+
+Click the color marker icon unveils a color legend to the right of the filter options. For each option, you can configure a custom color (default color: green).
```
Comment (MariekeGO) [inline user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Visualizations/Available_visualizations/Feeds/DashboardQueryFilterFeed.md:86]:
```suggestion
Click the color marker icon to see a color legend to the right of the filter options. For each option, you can configure a custom color (default color: green).
```

**Review by MariekeGO — CHANGES_REQUESTED** [review]
